internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json

```diff
@@ -4990,9 +4990,13 @@
         "Count_MortalityEvent_YearsUpto1_ICD10/P00-P96_AsAFractionOf_Count_Person_YearsUpto1",
         "Count_MortalityEvent_YearsUpto1_ICD10/Q00-Q99_AsAFractionOf_Count_Person_YearsUpto1",
         "Count_Person",
+        "Count_Person_0To4Years_AAC_FEMALE",
```

Comment:
Not a blocker, but I don;t see these Statvars in the schema repo. 

Can we check which import this is coming from?

Reply:
This is coming from the US Census PEP import. Will check the recent update CL changes/